dali/pipeline/data/tensor.h

```diff
@@ -422,6 +422,10 @@ class Tensor : public Buffer<Backend> {
     return *this;
   }
 
+  DALIMeta &GetMeta() {
+    return meta_;
+  }
```

Comment:
Do we really need this? It seems quite dangerous. I know we do have a standalone DALIMeta object, but setting the metadata for a tensor/batch can benefit from extra validation (e.g. against dimensionality/size/type/...). Returning a non-const reference to DALIMeta makes such extra validation impossible.

Reply:
Removed. As I said, I wanted to direct all calls through the DALIMeta api rather than repeat it in the TensorVector, but I am doing exactly the opposite with the access to tensors, so it doesn't make sense.